src/marqo/marqo_url_resolver.py

```diff
@@ -0,0 +1,29 @@
+import time
+
+import requests
+
+
+class MarqoUrlResolver:
+    def __init__(self, api_key=None, expiration_time: int = 60):
+        """ URL Resolver is a cache for urls that are resolved to their respective indices only for marqo cloud. """
+        self.timestamp = time.time() - 60
```

Comment:
consider `self.timestamp = time.time() - expiration_time ` if we consistently want to refresh the URLs after initialisation